srml/democracy/src/lib.rs

```diff
@@ -120,19 +129,32 @@ decl_module! {
 
 		/// Vote in a referendum. If `vote.is_aye()`, the vote is to enact the proposal;
 		/// otherwise it is a vote to keep the status quo.
+		///
+		/// # <weight>
+		/// - Weight depends on Self::do_vote
```

Comment:
Format should be consistent (i.e. end with a period, function names should be `preformatted`).
```suggestion
		/// - Weight depends on `Self::do_vote`.
```